atlas/templates/Makefile.vars.gotmpl

```diff
@@ -3,15 +3,17 @@ BUILD_PATH              := bin
 DOCKERFILE_PATH         := $(CURDIR)/docker
 
 # configuration for image names
+BUILD_NUMBER            ?= 0
 USERNAME                := $(USER)
-GIT_COMMIT              := $(shell git describe --dirty=-unsupported --always --tags || echo pre-commit)
-IMAGE_VERSION           ?= $(GIT_COMMIT)-$(USERNAME)
+GIT_COMMIT              := $(shell git describe --dirty=-unsupported --always --tags --long || echo pre-commit)
+IMAGE_VERSION           ?= $(GIT_COMMIT)-j$(BUILD_NUMBER)
```

Comment:
Should this just be overridden in the Jenkinsfile, since it's meaningless on local builds? (although I might support removing the `-$(USERNAME)` suffix here.

Reply:
sure